pkg/apis/apps/install/install.go

```diff
@@ -37,7 +37,7 @@ func Install(groupFactoryRegistry announced.APIGroupFactoryRegistry, registry *r
 	if err := announced.NewGroupMetaFactory(
 		&announced.GroupMetaFactoryArgs{
 			GroupName:                  apps.GroupName,
-			VersionPreferenceOrder:     []string{v1beta1.SchemeGroupVersion.Version, v1beta2.SchemeGroupVersion.Version},
+			VersionPreferenceOrder:     []string{v1beta2.SchemeGroupVersion.Version, v1beta1.SchemeGroupVersion.Version},
```

Comment:
@janetkuo This is just the preferences order. It will not break anything < 1.8 kubectl.